tests/unit/objects/test_projects.py

```diff
@@ -682,29 +661,6 @@ def test_delete_project(project, resp_delete_project):
     project.delete()
 
 
-def test_upload_file(project, resp_upload_file_project):
-    project.upload("filename.png", "raw\nfile\ndata")
-
-
-def test_upload_file_with_filepath(project, resp_upload_file_project):
-    with patch("builtins.open", mock_open(read_data="raw\nfile\ndata")):
-        project.upload("filename.png", None, "/filepath")
-
-
-def test_upload_file_without_filepath_nor_filedata(project):
-    with pytest.raises(
-        exceptions.GitlabUploadError, match="No file contents or path specified"
-    ):
-        project.upload("filename.png")
-
-
-def test_upload_file_with_filepath_and_filedata(project):
-    with pytest.raises(
-        exceptions.GitlabUploadError, match="File contents and file path specified"
-    ):
-        project.upload("filename.png", "filedata", "/filepath")
```

Comment:
Could we perhaps keep these unit tests? That way we know we don't have any regressions later on. If we need any shared fixtures we can put them into our `conftest.py`.

Reply:
Sure, makes sense!
I wasn't sure if we also wanted unit tests for wiki uploads - it seems redundant (everything seems to get tested as is), but what do you think?